codeHF/config_tasks.sh

```diff
@@ -187,11 +188,13 @@ function AdjustJson {
     ReplaceString "\"processMc\": \"false\"" "\"processMc\": \"true\"" "$JSON" || ErrExit "Failed to edit $JSON."
     ReplaceString "\"processMC\": \"false\"" "\"processMC\": \"true\"" "$JSON" || ErrExit "Failed to edit $JSON."
     ReplaceString "\"isMC\": \"false\"" "\"isMC\": \"true\"" "$JSON" || ErrExit "Failed to edit $JSON."
+    ReplaceString "\"processData\": \"true\"" "\"processData\": \"false\"" "$JSON" || ErrExit "Failed to edit $JSON."
```

Comment:
I am not sure this is correct. Some workflows run `processData` also in MC.

Reply:
OK, then my proposed solution here is of course problematic. But at least some tree creators (including the lc-to-k0s-p one), need to run exclusively `processData` or `processMc`, so we need to switch that on/off.

Reply:
Then those devices should have `processData` set to `false` by default in the JSON.

Reply:
Ok, for the tree-creator-lc-to-k0s-p I have it already like this, for the tree-creator-lc-to-p-k-pi `processData` is currently set to `true` in the dpl-config_Run3.json. Should I change that too?
Did I get you correctly that then I should only remove the line
`ReplaceString "\"processData\": \"true\"" "\"processData\": \"false\"" "$JSON" || ErrExit "Failed to edit $JSON."`
and leave the opposite 
`ReplaceString "\"processData\": \"false\"" "\"processData\": \"true\"" "$JSON" || ErrExit "Failed to edit $JSON."`
in as it is now?